recipes/boto3/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "boto3" %}
+{% set version = "1.4.1" %}
+{% set sha256 = "f73644c495f24020f20d0f6c22b9b8c4bba13416a2a17ad27120a5a24ac52e3b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - botocore >=1.4.1,<1.5.0
+    - jmespath >=0.7.1,<1.0.0
+    - s3transfer >=0.1.0,<0.2.0
+
+test:
+  imports:
+    - boto3
+    - boto3.docs
+    - boto3.dynamodb
+    - boto3.ec2
+    - boto3.resources
+    - boto3.s3
+
+about:
+  home: http://aws.amazon.com/sdk-for-python/
+  dev_url: https://github.com/boto/boto3
+  doc_url: https://boto3.readthedocs.org
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Amazon Web Services SDK for Python
+  description: Boto3 makes it easy to integrate you Python application, library or script
+    with AWS services. It allows Python developers to write softare that makes
+    use of services like Amazon S3 and Amazon EC2.
+
+extra:
+  recipe-maintainers:
+    - proinsias
```

Comment:
Haven't heard back. So am going to go ahead and merge. Feel free to add yourself as a maintainer in the feedstock if you are interested.